efficientdet/inference.py

```diff
@@ -367,7 +367,17 @@ def build(self,
         max_boxes_to_draw=max_boxes_to_draw)
 
     if not self.sess:
-      self.sess = tf.Session()
+      if self.optimize:
+          config_proto = tf.ConfigProto(
+            allow_soft_placement=True, log_device_placement=False)
+          from tensorflow.core.protobuf import rewriter_config_pb2
+          config_proto.graph_options.rewrite_options.auto_mixed_precision = rewriter_config_pb2.RewriterConfig.ON
```

Comment:
```config_proto.graph_options.rewrite_options.auto_mixed_precision``` and 
config_proto.graph_options.rewrite_options.memory_optimization = ```rewriter_config_pb2.RewriterConfig.RECOMPUTATION_HEURISTICS``` could decrease memory usage

Reply:
I have added the complete mixed precision support: --hparams="precision=mixed_float16".
RECOMPUTATION_HEURISTICS can indeed reduce memory usage during training, but since this is for inference, I guess it doesn't have much value here?